masterlist.yaml

```diff
@@ -10426,6 +10426,667 @@ plugins:
       - 'Requiem - Hearthfire.esp'
       - 'Requiem - Height Adjusted Races with True Giants.esp'
       - 'NRM Dragonborn - Requiem Patch.esp'
+
+
+#Based on thetrader's loadorder ( http://www.nexusmods.com/skyrim/mods/61621/ )
+  - name: 'Requiem - Hearthfire.esp'
+    after:
+      - 'Requiem - Bugsmasher Edition.esp'
+      - 'Requiem - Legendary Bugsmasher Edition.esp'
+      - 'Requiem - No Messages.esp'
+      - 'Requiem - No Overencumbered Messages.esp'
+      - 'Requiem - Restored Rewarding Sounds.esp'
+      - 'Requiem - Restored Saving Messages.esp'
+      - 'Requiem - Vanilla Shrines.esp'
+
+#Based on thetrader's loadorder ( http://www.nexusmods.com/skyrim/mods/61621/ )
+  - name: 'Requiem - Audio Overhaul for Skyrim.esp'
+    after:
+      - 'Requiem - Bugsmasher Edition.esp'
+      - 'Requiem - Legendary Bugsmasher Edition.esp'
+      - 'Requiem - No Messages.esp'
+      - 'Requiem - No Overencumbered Messages.esp'
+      - 'Requiem - Restored Rewarding Sounds.esp'
+      - 'Requiem - Restored Saving Messages.esp'
+      - 'Requiem - Vanilla Shrines.esp'
+      - 'Requiem - Hearthfire.esp'
+
+	  
```

Comment:
There's not much point making another commit to correct only this, but in future please don't leave trailing whitespace. All good editors have an option to trim it for you automatically.